snapshot/snapshot.go

```diff
@@ -677,3 +683,34 @@ func (o *snapshotter) checkAvailability(ctx context.Context, key string) bool {
 	}
 	return true
 }
+
+func (o *snapshotter) restoreRemoteSnapshot(ctx context.Context) error {
+	mounts, err := mount.Self()
+	if err != nil {
+		return err
+	}
+	for _, m := range mounts {
+		if strings.HasPrefix(m.Mountpoint, filepath.Join(o.root, "snapshots")) {
+			if err := syscall.Unmount(m.Mountpoint, syscall.MNT_FORCE); err != nil {
+				return errors.Wrapf(err, "failed to unmount %s", m.Mountpoint)
+			}
+		}
+	}
+
+	var task []snapshots.Info
+	if err := o.Walk(ctx, func(ctx context.Context, info snapshots.Info) error {
+		if _, ok := info.Labels[remoteLabel]; ok {
+			task = append(task, info)
+		}
+		return nil
+	}); err != nil && !errdefs.IsNotFound(err) {
```

Comment:
Why is ErrNotFound ignored here?

Reply:
If the database is empty, it will return "ErrNotFound"